src/tests/functional/rendering_tests.js

```diff
@@ -580,6 +582,15 @@ test("rendering a redirect response replaces the body once and only once", async
   assert.ok(await noNextBodyMutation(page), "replaces <body> element once")
 })
 
+
+test.only("hides progress bar when reloading due to request failure", async ({ page }) => {
```

Comment:
Could you remove the `.only` call?

```suggestion
test("hides progress bar when reloading due to request failure", async ({ page }) => {
```